bluetooth/tracing/BluetoothStack.wprp

```diff
@@ -870,8 +912,15 @@
             <EventProviderId Value="Microsoft.Windows.Apps.PhoneShellUi"/>
             <EventProviderId Value="Microsoft.Windows.Apps.PhoneUtilsProviders"/>
             <EventProviderId Value="Microsoft.Windows.Apps.TelephonyInteractiveUser"/>
+            <EventProviderId Value="Microsoft.Windows.Apps.VoipRTProviders"/>
+            <EventProviderId Value="Microsoft-Windows-Telephony-VoipRT"/>

```

Comment:
Why are you adding our VoIP providers to the Bluetooth .wprp? This should just be for HF calling right?

Reply:
Our phoneSvc support Voip calling also, and we've seen issue before that affect voip calling stack when interop with HF, so we can use the same trace profile to debug those kind of issue.